tests/service_atcoder.py

```diff
@@ -282,7 +282,8 @@ def test_get_score_float(self):
 
 
 class AtCoderProblemDataTest(unittest.TestCase):
-    def test_from_html(self):
+    # test the third format (stated in AtCoderProblemDetailedData)
```

Comment:
If you apply my comment about xth format, These tests order become "first format, second format and third format".
It seems more natural.